input/intro-notes/StructureDefinition-therapeutic-implication-intro.xml

```diff
@@ -1,12 +1,8 @@
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir ../../src-generated/schemas/fhir-single.xsd">
-  <blockquote class="stu-note">
-    <p>
-    The content in this section may be significantly revised prior to the next ballot after initial use and feedback.
-    </p>
-	  
-  </blockquote>
   <p>
-    This section provides guidance for genomic reporting about the implication of a patient's genetics on the behavior of one or more medications.  This includes making recommendations for medication adjustments.  This portion of the implementation guide relies on the content in the <a href="general.html">General Genomic Reporting</a> and <a href="sequencing.html">Variant Reporting</a> portions of this implementation guide.  Pharmacogenomic reports supplement this information with a set of pharmacogenomic-specific implication profiles. Implementers of pharmacogenomic reporting may also be interested in the <a href="somatics.html">Somatic Reporting</a> section of this implementation guide.
+    This section provides guidance for genomic reporting about the implication of a patient's genetics on the behavior of one or more 
```

Comment:
NIT: formatting is off again (it probably renders OK, but does look odd in the source).

Reply:
fixed